src/Components/Publishing/Ads/AllowedUsers.ts

```diff
@@ -0,0 +1,29 @@
+import { get } from "lodash"
+import { data as sd } from "sharify"
+
+/**
+ * For now ads, served by Hashtag Labs are only to viewable in the following instances:
+ *  1) When the environment is NOT Production
+ *  2) When a user is an Admin
+ *  3) When a user has been added to the allowlist set via HASHTAG_LAB_ADS_ALLOWLIST
+ */
+
+export const getAllowedUsers = () => {
+  const allowedUsers = (sd.HASHTAG_LAB_ADS_ALLOWLIST || "")
+    .split(",")
+    .filter(Boolean)
+  const currentUser = get(sd, "CURRENT_USER.email", "")
+  const env = sd.HASHTAG_LAB_ADS_ENABLED
```

Comment:
This is a bit ambiguous naming-wise; things like "enabled", "disabled", "on", "off" all all reduced in practice to booleans values -- true / false. 

Since ENV vars are set on the server _per environment_ it's totally possible to set this to  `true` on staging and `false` production, and then simplify things down below with 

```js
if (sd.HASHTAG_LAB_ADS_ENABLED) {
  ...
}
```